docs/guides/advanced/ServerRendering.md

```diff
@@ -53,7 +53,7 @@ render(<Router history={history} routes={routes} />, mountNode)
 You need to do
 
 ```js
-match({ routes, location }, (error, redirectLocation, renderProps) => {
-  render(<Router {...renderProps} history={history} />, mountNode)
+match({ history, routes }, (error, redirectLocation, renderProps) => {
```

Comment:
Circling back up on this – thoughts? In 3.0, the requirement ought to be either a `history` _or_ a `matchContext`.